scripts/sct_dmri_concat_b0_and_dwi.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+#
+# Merge b=0 and dMRI data and output appropriate bvecs/bvals files.
+#
+# Copyright (c) 2019 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Author: Julien Cohen-Adad
+#
+# About the license: see the file LICENSE.TXT
+
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+
+from dipy.data.fetcher import read_bvals_bvecs
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+from spinalcordtoolbox.image import Image, concat_data
+
+import sct_utils as sct
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description="Concatenate b=0 scans with DWI time series and update the bvecs and bvals files. Note that you can"
+                    "concatenate more than two files (e.g.: b0 dwi1 dw2 dw3).",
```

Comment:
> Note that you can concatenate more than two files (e.g.: b0 dwi1 dw2 dw3).

Was wondering if we should move this information to the `-i` help!?

(+ small typo for `dw2` and `dw3`)

Reply:
good point-- fixed in 3fd4b98aa23ee806ce15d9dfc7c7df4e02251a8e